crypto/bio/bss_conn.c

```diff
@@ -484,8 +496,21 @@ static long conn_ctrl(BIO *b, int cmd, long num, void *ptr)
         else
             data->connect_mode &= ~BIO_SOCK_NONBLOCK;
         break;
+    case BIO_C_SET_TFO:
```

Comment:
I'm not sure this should happily return 1 even if TCP_FASTOPEN is not defined and thus not supported.

Reply:
Yeah, if the platform doesn't support TCP_FASTOPEN, this should be ignored.